### ISSUE TITLE
2.49RC2: "Check settings at startup" is not working 

### ISSUE THREAD

**Issue by yesgrey** [gitextensions/gitextensions#3404]
Hi,

The "Check settings at startup" option seems to not be working. If I start Gitextensions and no user name or email address are set, gitextensions starts normally and do not show up automatically the settings dialog showing in red the settings which are not valid. If I then go to settings, it still shows correctly that some settings are nor correct, so, maybe something is not working correctly during the startup phase?...

Regards

**Comment by KindDragon** [comment]
Sorry, cannot reproduce

**Comment by yesgrey** [comment]
Ok, that's not good...

I've just tried again with a simple case.

Started from a good configuration where all the settings shown green. Edited the settings by deleting the username. Closed and then restarted Git extensions.

With 2.48.05, whenever this happened Git extensions would automatically open the "Settings - Checklist" dialog showing up that one of the settings was wrong and that it should be corrected. Now it doesn't, it opens normally showing Gitextensions main window, like if everything was fine.

I've tried this in two machines, one with Windows 7 and another with Windows 10. I hope this can be helpful, otherwise I don't know what to do more...

**Comment by KindDragon** [comment]
You sure that checkbox "Check settings at startup" is checked?

**Comment by yesgrey** [comment]
Yes, it's checked.

By the way, are you considering releasing a new release (maybe RC3) soon? It would be really nice to have at least the other small bugs fixed... ;) I tried to build one myself but I'm having some problems related with some of the submodules not being cloned...

**Comment by KindDragon** [comment]
We released final 2.49 https://github.com/gitextensions/gitextensions/releases/tag/v2.49

**Comment by yesgrey** [comment]
OK, great! I think this issue is not important enough to delay its release. I will try with the official 2.49, maybe it's working fine with it now.

**Comment by yesgrey** [comment]
I have tried with the 2.49 but still doesn't work.

If you or anyone else cannot reproduce it, then I don't know what to do more...

**Comment by vbjay** [comment]
If I remember, the check box gets turned off when the settings are all ok.

**Comment by yesgrey** [comment]
Yes, but I turn it on, and when Git extensions starts it doesn't turn it off because it detects that some settings are missing, however it doesn't open the "Settings - Checklist" dialog.

**Comment by RussKie** [comment]
Is this still an issue?

**Comment by yesgrey** [comment]
Yes, still happening.
